app/services/case_linking_service.rb

```diff
@@ -34,6 +34,34 @@ def call
     @result = :error
   end
 
+  def destroy
+    ActiveRecord::Base.transaction do
+      # find the linked case
+      @link_case = Case::Base.where(number: @link_case_number).first
```

Comment:
I think `@case.linked_cases.find_by(number: @link_case_number).first` would make more sense and be safer here.

Reply:
👍